Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <team.coretools@kisio.com>", "Guillaume Pinot <texitoi@texitoi.eu>"]
 name = "transit_model"
-version = "0.32.2"
+version = "0.32.3"
```

Comment:
I would do a `0.33.0` since the only PR you'll be including is changing the behavior of the GTFS export.